shared/actions/favorite.js

```diff
@@ -9,12 +9,14 @@ import type {UserList} from '../common-adapters/usernames'
 import {NotifyPopup} from '../native/notifications'
 import {apiserverGetRpc, favoriteFavoriteAddRpc, favoriteFavoriteIgnoreRpc} from '../constants/types/flow-types'
 import {badgeApp} from './notifications'
-import {canonicalizeUsernames, parseFolderNameToUsers} from '../util/kbfs'
+import {parseFolderNameToUsers, sortUserList} from '../util/kbfs'
 import {defaultKBFSPath} from '../constants/config'
 import {navigateBack} from '../actions/router'
 
 export function pathFromFolder ({isPublic, users}: {isPublic: boolean, users: UserList}) {
-  const sortName = users.map(u => u.username).join(',')
+  const rwers = users.filter(u => !u.readOnly).map(u => u.username)
+  const readers = users.filter(u => !!u.readOnly).map(u => u.username)
```

Comment:
I know I can use `_.partition` but didn't want to import it, if there's demand I'll change it